source/graphql/authenticate.txt

```diff
@@ -15,130 +13,119 @@ Authenticate GraphQL Requests
 Overview
 --------
 
-The Atlas GraphQL API operates over HTTP, which means that you can access your
-:ref:`exposed data <graphql-expose-data>` using any HTTP or GraphQL client.
-Atlas App Services enforces rules for all GraphQL operations, so any GraphQL HTTP request
-must include an application user's login credentials or a valid access token.
+The GraphQL API requires that incoming requests include authentication
+information for the user making the request. This lets the API enforce
+rules and validate document schemas for each operation.
```

Comment:
FYI - past this opening paragraph this page is _mostly_ a straight copy-paste port of the Data API page with GraphQL URLs & examples instead of Data API.

Reply:
thx for heads up. reading thru but won't repeat any comments.

Reply:
no comments specific to this page. ditto all relevant comments on the Data API page.